### DISCUSSION TITLE
Can I check the permissions of a fine-grained (beta) token?

### DISCUSSION THREAD

**Discussion by dmartinochoa** [community/community#55914] [Other Feature Feedback, Questions, & Ideas]
Question

Is there a simple way to check the permissions of a fine-grained token using the github API?

The solution provided in a [previous similar discussion](https://github.com/orgs/community/discussions/25259#discussioncomment-3247158) only seems to work for classic tokens:
```
$ curl -sS -f -I -H "Authorization: token ghp_yourOwnToken" https://api.github.com 
```
But the response when using a fine-grained token in this request does not seem to have the data needed.

**Comment by 4chref** [comment]
To check the permissions associated with a fine-grained token using the GitHub API, you can use the "GET /applications/{client_id}/tokens/{access_token}" endpoint.

Here's how you can do it using cURL:

```
$ curl -H "Authorization: Bearer YOUR_ACCESS_TOKEN" https://api.github.com/applications/{client_id}/tokens/{access_token}
```
